cmd/builder/internal/builder/templates/go.mod.tmpl

```diff
@@ -34,6 +34,12 @@ require (
 	github.com/knadh/koanf/providers/confmap v0.1.0 // indirect
 )
 
+{{- range .Providers}}
+{{if ne .Path ""}}replace {{.GoMod}} => {{.Path}}{{end}}
+{{- end}}
+{{- range .Converters}}
+{{if ne .Path ""}}replace {{.GoMod}} => {{.Path}}{{end}}
+{{- end}}
```

Comment:
Can you do a separate PR and rename .Providers -> .ConfmapProviders same for converter?

Reply:
yeah that makes sense to me

Reply:
do you want to wait to merge this one in until that PR is open? seems like it would make sense to merge this and then I can create a new branch from main with this included